pv_site_api/_db_helpers.py

```diff
@@ -229,3 +231,36 @@ def check_user_has_access_to_sites(session: Session, auth: dict, site_uuids: lis
                     f"does not have access to this site {site_uuid}. "
                     f"User has access to {site_uuids}",
                 )
+
+
+def get_sites_from_user(session, user, latitude_longitude_max=None, latitude_longitude_min=None):
```

Comment:
hi @peterdudfield, just a general question here, why don't functions like this one live in the `pv site datamodel`? for instance, if it did live in the data model (or some version of it, i realise the `LatLonLimits` class is particular to the api), then this function could be available for the OCF Dashboard as well.

Reply:
short answer is: It should be in `pv site datamodel`, Ill move it over

Reply:
I've put it here - https://github.com/openclimatefix/pv-site-datamodel/issues/83